credentials/service.js

```diff
@@ -84,10 +84,11 @@ function clearRemoteData (hook) {
 }
 
 function setProfileData (hook) {
+  // TODO: IK: fairly certain I added the email field in here for simpler population of Cobuy profiles which have emails, if so that was a mistake as profile migration of this repo has no email field
```

Comment:
i'm not sure i follow... but anyways, can be cleaned up when we merge the tables. :smiley: